src/main/java/org/sonarqube/A.java

```diff
@@ -3,7 +3,11 @@
 public class A {
   String message = "foo";
 
+  // TODO fix this
   public String foo() {
+    message = null;
+    message.length();
+
     return message;
```

Comment:
###AWAMlBG2b0Hum1nhz4KJ###
org.sonarqube:pr-decoration:src/main/java/org/sonarqube/A.java:BRANCH:pr_8@9
A "NullPointerException" could be thrown; "message" is nullable here.

Reply:
simulating auto reply to comment

Reply:
simulating 2nd auto reply to comment

Reply:
simulating non auto reply to comment